content/en/docs/faq/installation/uninstall-pluggable-components.md

```diff
@@ -0,0 +1,218 @@
+   done
+   
+   for s2iruns in `kubectl get s2iruns -A -o jsonpath="{.items[*].metadata.name}"`
+   do
+     kubectl patch s2iruns $s2iruns -n `kubectl get s2iruns -A | grep $s2iruns | awk '{print $1}'` -p '{"metadata":{"finalizers":null}}' --type=merge
+   done
+   
+   kubectl delete devopsprojects --all 2>/dev/null
+   ```
+
+   ```bash
+   kubectl delete ns kubesphere-devops-system
+   ```
+
+## Uninstall KubeSphere Logging
+
+1. Change the value of `logging.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
+
+2. To disable only log collection:
+
+   ```bash
+   delete inputs.logging.kubesphere.io -n kubesphere-logging-system tail
+   ```
+
+   {{< notice note >}}
+
+   After running this command, you can still view the container recent logs provided by Kubernetes by default. However, the container history logs will be cleared and you cannot browse them any more. 
+
+   {{</ notice >}}
+
+3. To uninstall Logging system including Elasticsearch:
+
+   ```bash
+   kubectl delete crd fluentbitconfigs.logging.kubesphere.io
+   kubectl delete crd fluentbits.logging.kubesphere.io
+   kubectl delete crd inputs.logging.kubesphere.io
+   kubectl delete crd outputs.logging.kubesphere.io
+   kubectl delete crd parsers.logging.kubesphere.io
+   kubectl delete deployments.apps -n kubesphere-logging-system fluentbit-operator
+   helm uninstall elasticsearch-logging --namespace kubesphere-logging-system
+   ```
+
+   {{< notice note >}}
+
+   This operation may cause anomalies in Auditing, Events, and Service Mesh.
+
+   {{</ notice >}}
+
+## Uninstall KubeSphere Events
+
+1. Change the value of `events.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
+
+2. Run the following command:
+
+   ```bash
+   helm delete ks-events -n kubesphere-logging-system
+   ```
+
+## Uninstall KubeSphere Alerting
+
+1. Change the value of `alerting.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
+
+2. Run the following command:
+
+   ```bash
+   kubectl -n kubesphere-monitoring-system delete thanosruler kubesphere
+   ```
+
+   {{< notice note >}}
+
+   Notification is installed in KubeSphere v3.1.x by default, so you do not need to uninstall it.
+
+   {{</ notice >}} 
+
+
+## Uninstall KubeSphere Auditing
+
+1. Change the value of `auditing.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
+
+2. Run the following commands:
+
+   ```bash
+   helm uninstall kube-auditing -n kubesphere-logging-system
+   kubectl delete crd awh
+   kubectl delete crd ar
+   ```
+
+## Uninstall KubeSphere Service Mesh
+
+1. Change the value of `servicemesh.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
+
+2. Run the following commands:
+
+   ```bash
+   curl -L https://istio.io/downloadIstio | sh -
+   istioctl x uninstall --purge
+   
+   kubectl -n istio-system delete kiali
```

Comment:
The delete command should be:
`kubectl -n istio-system delete kiali kiali`

Reply:
Thanks for your comment. Updates made in [this commit](https://github.com/kubesphere/website/pull/1863/commits/b1d0432ebb9a06e07d329f6aa4c3b3bc04a70f8c).